_includes/jumbotron.html

```diff
@@ -7,5 +7,10 @@ <h2 itemprop="description">The pluggable linting <span itemprop="applicationCate
           </div>
           <div class="col-md-4 col-sm-4"></div>
         </div>
+        <div class="row">
```

Comment:
I can't say I love the placement of that button, but I can't really come up with the alternative...